From a6b389c2bd4a5ad5ad483179c4cbcd2ef5a0e66a Mon Sep 17 00:00:00 2001
From: Sreedhar Govardhana <srgovard@amd.com>
Date: Thu, 29 Jul 2021 19:08:16 -0500
Subject: [PATCH] ARM:dts:aspeed: Add I3C entries to Onyx dts file

Adding I3C entries for DIMM SPD and APML to the Onyx dts.

Signed-off-by: Sreedhar Govardhana <srgovard@amd.com>
---
 arch/arm/boot/dts/aspeed-bmc-amd-onyx.dts | 21 +++++++++++++++------
 1 file changed, 15 insertions(+), 6 deletions(-)

diff --git a/arch/arm/boot/dts/aspeed-bmc-amd-onyx.dts b/arch/arm/boot/dts/aspeed-bmc-amd-onyx.dts
index b997469fed49..f45c6a5ade3b 100644
--- a/arch/arm/boot/dts/aspeed-bmc-amd-onyx.dts
+++ b/arch/arm/boot/dts/aspeed-bmc-amd-onyx.dts
@@ -132,12 +132,6 @@ tca9546a@70 {
 	};
 };

-&i2c2 {
-	// Net name i3c4 - P0 APML
-	// Will use i2c until I3C is enabled/verified on SoC
-	status = "okay";
-};
-
 &i2c4 {
 	// Net name i2c1
 	status = "okay";
@@ -349,6 +343,21 @@ adc121c@55 {
 	};
 };

+&i3c0 {
+	// P0 DIMM (A-F) SPD Access
+	status = "okay";
+};
+
+&i3c1 {
+	// P0 DIMM (G-L) SPD Access
+	status = "okay";
+};
+
+&i3c4 {
+	// P0 APML
+	status = "okay";
+};
+
 &kcs1 {
 	status = "okay";
 	aspeed,lpc-io-reg = <0x60>;
--
2.25.1

